packages/connect/src/connect.js

```diff
@@ -1,140 +1,193 @@
 import {
-  Component,
-  createContext,
-  createElement,
-  memo,
-  useMemo,
   useState,
   useEffect,
-  useContext,
+  useMemo,
+  memo,
   useRef,
+  createContext,
+  useContext,
 } from "react";
 import { warn } from "@frontity/error";
-import { observe, unobserve, raw, isObservable } from ".";
+import { observe, unobserve } from "@frontity/observer-util";
```

Comment:
I wasn't sure where to comment this but I see that we're not using the `view` function from `@frontity/react-easy-state` and instead we're creating our own wrapper here. 

Would it make sense to delete it altogether from our fork to shave some bytes (let's make @cristianbote happy 😅 )? I don't think that the function gets tree-shaken out of the bundle, right?

Reply:
No, I don't think it will get tree shaken because due to the scheduler, `react-easy-state` contains side-effects. 

I'll remove it from our fork 👍

Reply:
The thing is that if I remove `view` I'll kind of break the library because tests won't pass. I'd have to remove the tests, but I think it'll be easier in the future if we need to do a change to make sure that everything works in `react-easy-state`.

Maybe we could mark the library as tree-shakable, except for the scheduler. 

@cristianbote, I've never dealt with this type of side-effect configuration before. Could you please help me out with that? Is it possible to do it, right?

Reply:
We can make that in a different PR to avoid blocking this one (it's actually a different repo).

So I'll resolve this for now 🙂

Reply:
yeah, I didn't mean this to be a blocker of course. We can do it in a future PR :)